sections/introduction.tex

```diff
@@ -1,8 +1,13 @@
 \section{Introduction}\label{sec:introduction}
+% Intro to meta data
+Data is everywhere and its used in many different scenarios and use cases. 
+When referring to a data set, we talk about a data having meta-data information or attributes, which is the entries that could describe certain features about the data, like petal length from well known IRIS data set\footnote{https://archive.ics.uci.edu/ml/datasets/iris} describing the length of a particular flowers petals.
+However, there is usually a lot of data, that is not used, either because it is not useful or we do not know how to use it properly.
```

Comment:
```suggestion
Data is everywhere and it is used in many different scenarios and use cases. 
When referring to a data set, we talk about data having meta-data information or attributes, which is the entries that could describe certain features about the data, like petal length from well-known IRIS data set\footnote{https://archive.ics.uci.edu/ml/datasets/iris} describing the length of a particular flower's petals.
However, there is usually a lot of data, that is not used, either because it is not useful or we do not know how to use it properly.
```